### ISSUE TITLE
Breadcrumb/Breadcrumb Item Review Notes

### ISSUE THREAD

**Issue by Darper** [DigitalRiver/react-atlas#966]
**Documentation**
- Add margin to the right of the Avatar in the 2nd example

**Code**
- Create new prop for custom delimiter (don't force `/`)
- Remove `/` delimiter from CSS and put into JS
- If aria-label is needed, change to "Breadcrumb"
- Check breadcrumb examples on W3 to identify missing accessibility tags. Update HTML if necessary. (https://www.w3.org/TR/wai-aria-practices/#breadcrumb)
- Remove defaultProps
- Change `active` to apply both styling and `aria-current="page"` attribute. Research aria-current to verify usage and testing strategy. If possible, place on BreadcrumbItem `<li>` tag so that user doesn't have to worry about adding it themselves.
- Add prop spread `...others` to both components and pass to rendered HTML.
- Improve prop description for `active`.

**Comment by stuller** [comment]
bah tried to change to a checklist but it didn't work
